src/pages/index.js

```diff
@@ -93,34 +254,62 @@ const cardSection = new Section(
   },
   cardClassSelector
 );
+*/
 
 const imagePopup = new PopupWithImage({ popupSelector: previewModalSelector });
 
 const addDestinationPopup = new PopupWithForm({
   popupSelector: addDestinationSelector,
   handleFormSubmit: (formData) => {
     console.log(formData);
-
+    console.log(formData);
     formData.link = formData.destination_image_URL;
     delete formData.destination_image_URL;
 
     formData.name = formData.destination_title;
     delete formData.destination_title;
+    api
+      .addNewCard(formData)
+      .then(() => {
+        //if (res.ok) {
 
-    cardSection.addItem(formData); //BULLET POINT #12 RESOLUTION
-    //test
+        return api
+          .getCards()
```

Comment:
Please, don't call `api.getCards()` when you add a new card because you get the card object in the response when you call `api.addNewCard(formData)`